src/code42cli/cmds/search/extraction.py

```diff
@@ -61,12 +69,22 @@ def handle_response(response):
                 events = _get_alert_details(sdk, events)
             except Exception as ex:
                 handlers.handle_error(ex)
+
         handlers.TOTAL_EVENTS += len(events)
-        event = None
-        for event in events:
-            output_logger.info(event)
-        if event:
-            last_event_timestamp = extractor._get_timestamp_from_item(event)
```

Comment:
I would like to restructure code here, instead of processing events here in the handler, they should be processed in their respective "feature" module. 
This will allow us to implement separate processing of `security-data` events and `alert` events if needed. Also with that lines 67 to 71 can move in `alerts` module.  

Implying, let this module be generic and maintain separation of concern, without being aware what type of data it is processing.

This can be achieved by: `handlers.EVENTS = events` without needing a return statement.